tests/foreman/destructive/test_backup_restore.py

```diff
@@ -24,7 +24,7 @@
 from robottelo.config import settings
 from robottelo.logging import logger
 
```

Comment:
@omkarkhatavkar As this module does not use any common fixtures/module_target_sat and uses sat_maintain specific fixtures, could it be ignored for moving?

Reply:
Plan was to move all destructive tests in destructive folder. This is common pattern we wanted to have. Having said that you are correct functionally it will work. But can not set inconsistent pattern.

Reply:
Why do you see that you need to change the CI job here? what are the problems are you facing ? 

We wanted to move to destructive so it can be used in pytest -component <component_name>. Are you not running your tests using `pytets -component <name>` pattern ? 

I would also even ask you a question here, are these tests you mentioned are qualify for destructive tests?  If not then no need to put them in destructive folder.

Reply:
@Gauravtalreja1 you can still create a new directory for maintain tests, but I'd think it would just have a module in the `tests/foreman/cli` directory.  Destructive tests, just like all other components, will be split into a module in the destructive folder.  No CI job updates should be needed for component pipelines, they're collecting everything in `tests/foreman`, and the destructive tests still have a component set.